core/src/main/java/hudson/Util.java

```diff
@@ -507,39 +511,23 @@ public static boolean isSymlink(@Nonnull File file) throws IOException {
          *  both as symlinks, let's use that function and always call it before calling down to the
          *  NIO2 API.
          *
+         *  JENKINS-39179, JENKINS-36088.
```

Comment:
Probably some of the comment lines could be deleted/reworked at this point. Also the `Taken from …` comment on line 500 is probably now obsolete.